modules/swagger-core/src/main/java/io/swagger/jackson/ModelResolver.java

```diff
@@ -1,3 +1,19 @@
+/**
+ * Copyright 2017 SmartBear Software
+ * (c) Copyright 2017 Hewlett Packard Enterprise Development LP
```

Comment:
This is going to be tricky.

Reply:
Hi @webron, please could you elaborate on the problem here?

Reply:
By donating the code, the ownership is not going to be HPE's.

Reply:
I've been advised that if I wrote a portion of the code, then that portion is owned by my employer.  They are happy to contribute the code to your project, but feel that an appropriate copyright notice needs to be present for two reasons:
      * Regardless of whether the copyright line is included, the code I wrote is really owned by them.
      * Not notifying other potential users of the complete copyright ownership can only lead to misunderstandings down the road.